src/raftstore/store/worker/apply.rs

```diff
@@ -490,28 +499,54 @@ fn should_write_to_engine(cmd: &RaftCmdRequest, wb_keys: usize) -> bool {
     false
 }
 
+/// The apply deletgate of a Region which is responsible to handle committed
+/// raft log entries of a Region.
+///
+/// `Apply` is a term of Raft, which means executing the actual commands.
+/// In Raft, once some log entries are committed, for every peer of the Raft
+/// group will apply the logs one by one. For write commands, it does write or
+/// delete to local engine; for admin commands, it does some meta change of the
+/// Raft group.
+///
+/// `Delegate` is just a structure to congregate all apply related fields of a
+/// Region. The apply worker receives all the apply tasks of different Regions
+/// located at this store, and it will get the corresponding apply delegate to
+/// handle the apply task to make the code logic more clear.
 #[derive(Debug)]
 pub struct ApplyDelegate {
-    // peer_id
+    /// the ID of the peer
     id: u64,
-    // peer_tag, "[region region_id] peer_id"
+    /// the term of the Region
+    term: u64,
+    /// the Region information of the peer
+    region: Region,
+    /// peer_tag, "[region region_id] peer_id"
     tag: String,
+    /// the local engine to write kv and raft data
     engines: Engines,
-    region: Region,
-    // if we remove ourself in ChangePeer remove, we should set this flag, then
-    // any following committed logs in same Ready should be applied failed.
+
+    /// It is set to true when removing ourself because of `ConfChangeType::RemoveNode`, and then
```

Comment:
```suggestion
    /// Set to true when removing our self because of a `ConfChangeType::RemoveNode`, and then
```

Reply:
Can you tell me what is the difference between `ourself` with `our self`

Reply:
And this comment, I think, should be changed into `itself`